dripline/implementations/key_value_store.py

```diff
@@ -0,0 +1,19 @@
+from dripline.core import calibrate
+from dripline.core import Entity
+__all__ = []
+
+__all__.append("KeyValueStore")
+class KeyValueStore(Entity):
+
+    def __init__(self, initial_value=None, **kwargs):
+        Entity.__init__(self, **kwargs)
+        self._value = initial_value
+
+    @calibrate()
```

Comment:
nitpicky point: do we want brackets here (if it matters)? I think convention is not to use them unless an argument is passed. caught a double take from me

Reply:
we discussed on slack, but for anyone else looking at this: if a decorator takes arguments you have to call it as a function, even if you're using the decorator with only default values and nothing passed.